numba_dppy/examples/debug/commands/break_file:func

```diff
@@ -0,0 +1,17 @@
+# Run: gdb-oneapi -q -command commands/break_func:func python
```

Comment:
# Run: NUMBA_OPT=0 gdb-oneapi -q -command commands/break_func:func python

Reply:
NUMBA_OPT is not needed for breakpoints